docs/concepts/notifications.md

```diff
@@ -45,6 +46,57 @@ It's essential to utilize `secret_params` exclusively for handling sensitive inf
 
 See {py:class}`~mlrun.common.schemas.notification.NotificationKind`.
 
+## Mail notifications
+To send mail notifications, you need to use existing SMTP server.
+```python
+mail_notification = mlrun.model.Notification(
+    kind="mail",
+    when=["completed", "error", "running"],
+    name="mail-notification",
+    message="",
+    condition="",
+    severity="verbose",
+    params={
+        "start_tls": True,
+        "use_tls": False,
+        "validate_certs": False,
+        "email_addresses": ["hegeg56369@inikale.com"],
```

Comment:
What is this?
better to use a more generic example , like `user.name@domain.com`